userguide/one_shot_object_detection/README.md

```diff
@@ -44,34 +44,24 @@ model.save('mymodel.model')
 model.export_coreml('MyCustomOneShotDetector.mlmodel')
 ```
 
-Previewing the synthetic images generated for training is useful to gauge the kind of test images the model would be trained to handle. We can preview the augmented images generated and used by the model for training as follows:  
+Before starting the training process, you can preview the synthetic images the toolkit generates by calling the `preview_synthetic_training_data` utility. This helps you understand data the model would be trained on:  
```

Comment:
I would move this part to after how it works. It lacks context otherwise.

Reply:
Shall I move it in 'How it works'? Since we talk about generating synthetic images in that.

If we move to the next relevant section after it, that would be 'Advanced Usage - background images' where we talk about using custom background images and displaying our background images might be a bit confusing.

Reply:
You can move it to how it works. In that case, you don't need an advanced usage section.